docs/conf.py

```diff
@@ -28,6 +28,10 @@
 cwd = os.getcwd()
 project_root = os.path.dirname(cwd)
 
+# On Travis python3-gi is installed into folllowing directory, which
+# is not added to path because Travis uses virtualenv.
+sys.path.insert(0, '/usr/lib/python3/dist-packages')
```

Comment:
So the problem can be caused by the following:
1. Travis's Python creates a virtualenv with correctly detected site package directory.
2. In the created environment, tox creates another virtualenv with the Travis's Python in `sys.executable`, which is actually somehow symlinked to `/opt/python`.

The Travis uses Python 2.7, though, and only some files are symlinked to `/opt/python.` How does the leap from Python 2 to Python 3 actually happen?